### PR TITLE
More destructuring imports fix.

### PR DESCRIPTION
**Pull request by bowenni**
Fix https://github.com/angular/clutz/issues/642

### CODE REVIEW

**Comment by bowenni** [comment]
Failing tests are clutz/clutz tests not clutz/gents tests.

File: src/main/java/com/google/javascript/gents/ModuleConversionPass.java [hunk 1]
```diff
@@ -59,6 +60,10 @@
 
   private final String alreadyConvertedPrefix;
 
+  // Used to store destructuring assignments like "const {a, b} = abModule;".
+  // Later on we use this map to rewrite full module imports.
+  private final Map<String, Node> destructuringAssignments = new HashMap<>();
```
Comment (rkirov) [inline src/main/java/com/google/javascript/gents/ModuleConversionPass.java]: Can this be more specific than Node, I think you only use `ImportSpec` as values

File: src/main/java/com/google/javascript/gents/ModuleConversionPass.java [hunk 2]
```diff
@@ -598,7 +624,15 @@ private void convertRequireForAlreadyConverted(ModuleImport moduleImport) {
     } else if (moduleImport.isFullModuleImport()) {
       // const A = goog.require('...'); -> import * as A from '...';
       // It is safe to assume there's one full local name because this is validated before.
-      importSpec = Node.newString(Token.IMPORT_STAR, moduleImport.fullLocalNames.get(0));
+      String fullLocalName = moduleImport.fullLocalNames.get(0);
+      importSpec = Node.newString(Token.IMPORT_STAR, fullLocalName);
```
Comment (rkirov) [inline src/main/java/com/google/javascript/gents/ModuleConversionPass.java]: move this to an else branch

Comment (bowenni) [reply src/main/java/com/google/javascript/gents/ModuleConversionPass.java]: Done.

File: src/main/java/com/google/javascript/gents/ModuleConversionPass.java (hunk 1 above)
Comment (bowenni) [reply src/main/java/com/google/javascript/gents/ModuleConversionPass.java]: I'm not sure how to specify `ImportSpec` types here. `Node` does not take generic types.
I'm assuming that you meant to explicitly call out `ImportSpec` in the comments of this Map.

Comment (rkirov) [reply src/main/java/com/google/javascript/gents/ModuleConversionPass.java]: oh, forgot closure AST is untyped :/

**Pull request merged**